### ISSUE TITLE
Avatar integrate with react-native-fast-image

### ISSUE THREAD

**Issue by Alice-space** [GeekyAnts/NativeBase#5067]
### Description

It would be great to include react-native-fast-image in image-related components for better image experience

### Problem Statement

It would be great to include react-native-fast-image in image-related components for better image experience

### Proposed Solution or API

It would be great to include react-native-fast-image in image-related components for better image experience

**Comment by Viraj-10** [comment]
Hi @Alice-space, Thanks for the suggestion adding `react-native-fast-image` in native-base would add one more dependency to the library. if you are facing some issues with `Avatar` please let us know, we will try to fix it ASAP.

**Comment by Alice-space** [comment]
yeah, but I mean can you provide an interface to render image using custom image component instead of react native default one.

**Comment by focux** [comment]
I'd love this as well. It's okay for me if you don't support fast-image out of the box but it'd be good to have an interface to replace the image component for any we want.
